README.md

```diff
@@ -155,15 +155,18 @@ Then try `DawDreamer`:
 
 ### Linux
 
-Linux hasn't been tested since the transition from RenderMan, but it might work. There's also not yet a `Builds/Linux/make` file.
-
-Install [Python 3.8.x](https://www.python.org/downloads/source/).
-
-JUCE itself has a list of dependencies for Linux; it's a very big library - if you don't know it you should definitely take some time out to check it out! Depending on your distribution and setup you may already have some / all of the following libraries. If you are on Ubuntu, the following commands will install your dependencies. Find the respective packages for other distros using Google please!
+To build `DawDreamer` on Linux, you first need to install a few packages for JUCE.
+JUCE itself has a list of dependencies for Linux; it's a very big library - if you don't know it you should definitely take some time out to check it out!
+Depending on your distribution and setup you may already have some / all of the following libraries.
+If you are on Ubuntu, the following commands will install your dependencies:
 
 ```bash
-sudo apt-get -y install llvm
+sudo apt-get -y install ca-certificates
+sudo apt-get -y install build-essential
 sudo apt-get -y install clang
+sudo apt-get -y install pkg-config
+sudo apt-get -y install libboost-all-dev
+sudo apt-get -y install libboost-python-dev
```

Comment:
Just want to double check with you that boost is necessary? Maybe it's a JUCE thing?

Reply:
I also needed boost to get a renderman Dockerfile for linux: https://github.com/turian/renderman-dexed-linux/blob/main/Dockerfile